packages/core/src/State.ts

```diff
@@ -127,6 +127,7 @@ export class State<
    * An object mapping actor IDs to spawned actors/invoked services.
    */
   public children: Record<string, ActorRef<any>>;
+  public tags: Set<string>;
```

Comment:
`toJSON` should convert this to an array and an accompanying `.from` method (I think it's the one - but I might be wrong) should convert an array to a Set

Reply:
Done and test added